src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -37,6 +42,102 @@
             return assemblies;
         }
 
+        public bool ValidateExportUri(string outputFilePath, string exportUri)
```

Comment:
>bool [](http://example.com/codeflow?start=15&length=4)

Why is this an instance method? I don't see that you are operating against instance fields.

Reply:
thanks for the suggestion, when i first created it, i followed all the other methods (all were public).

Now, i'm changing to internal or internal static.

thanks for the suggestions :)